pom.xml

```diff
@@ -238,10 +238,18 @@
                         <groupId>org.springframework</groupId>
                         <artifactId>spring-messaging</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.springframework.batch</groupId>
+                        <artifactId>spring-batch-core</artifactId>
+                    </exclusion>
```

Comment:
@ibacher - I had to add in the above exclusion to get past the next error that @mogoodrich reported.  With this in place, I was able to successfully build/test, and to install the module into a 2.6.3 environment without any errors at startup.  Let me know what you think and hopefully this is ok and we can merge.

Reply:
Thanks @mseaton for that catch! This dependency management part is my least favorite part of the module.